extern/bundled/python-virtualenv/install.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail -x
+
+unset PYTHONPATH  # existing PYTHONPATH can interfere
+
+( # install a Python virtualenv
+if ! type virtualenv &>/dev/null; then
+    # if virtualenv isn't available, install it locally along with pip
+    [[ -e bootstrap/virtualenv.py ]] || {
+        curl -RLO https://bootstrap.pypa.io/get-pip.py &&
+        python get-pip.py virtualenv --ignore-installed --target bootstrap
```

Comment:
I guess it'll follow the system default at build time which may be undesirable. `python` also seems to get bundled as part of virtualenv but it's not exposed and only used for the pyhocon command. My intention for the bundled virtualenv was to serve only internal components but we might want to find a nice way to provide a default set of packages to the user. What's lacking here, I've been increasingly worried about, is a clear separation between the environment user's codes run in and deepdive's internal components do. We probably should try to not pollute user's env and document what exactly gets added to it, e.g., ddlib, psycopg2. Ideally users should be able to run deepdive in another virtualenv and their udfs shouldn't get interfered by the bundled virtualenv. Curious to hear any thoughts along the line!